LibVLCSharp.WinForms/LibVLCSharp.WinForms.csproj

```diff
@@ -31,7 +32,7 @@ LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.
   <ItemGroup>
     <ProjectReference Include="..\LibVLCSharp\LibVLCSharp.csproj" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='net40'">
```

Comment:
This Reference can be removed as well.

Reply:
Sorry guys.

I did a quick conversion to support netcoreapp3.0.

I should have probably spent more time understanding your build process. I used VS Code / PS Core terminal to compile the project. There are a lot of prerequisite VS 2017/19 payloads which I didn't want to install (UWP, Xamarin, etc). 

I was going to update the WPF sample app, but the project is still using old project format. 

Let me know if you want me to update the PR and remove the conditional references.

I would love a nuget package targeting netcoreapp3.0 ASAP :)

Reply:
we're on VS2017/VS4Mac 2017, but we need to make te migration anyway.
I guess we should make the migration all at once and see if any of the project has incompatibilities.

I might be able to give that a try in the weekend.

Reply:
I opened https://github.com/dashkan/libvlcsharp/pull/2 , but I just noticed Martin already did it...